core/src/main/java/com/datastax/oss/driver/api/core/config/DriverConfigProfile.java

```diff
@@ -79,10 +103,21 @@
 
   DriverConfigProfile withBytes(DriverOption option, long value);
 
+  List<Long> getBytesList(DriverOption option);
```

Comment:
This confused me initially (`bytes` represented as `long`).  But i see the comment on `getBytes` represents a size in bytes, not the `byte` type.

Reply:
I added a link to `getBytes`, and more explanations there.